js/main.js

```diff
@@ -23,5 +23,107 @@ function addingInputIntoObject(event) {
   data.entries.unshift(inputs);
   $imagePlaceHolder.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+  var $newLi = generateEntryDomTree(inputs);
+  var $ul = document.querySelector('ul');
+  $ul.prepend($newLi);
 }
 $form.addEventListener('submit', addingInputIntoObject);
+
+var $allViews = document.querySelectorAll('.view');
+function swappingViews(event) {
+  if (event.target.matches('.swap-view')) {
+    var $dataView = event.target.getAttribute('data-view');
+    for (var eachView = 0; eachView < $allViews.length; eachView++) {
+      $allViews[eachView].className = 'hidden';
+      if ($dataView === $allViews[eachView].getAttribute('data-view')) {
+        $allViews[eachView].classList.replace('hidden', 'view');
+        data.view = event.target.getAttribute('data-view');
+      }
+    }
```

Comment:
You should make this a separate function and make `$dataView` into the parameter, which can be the name of the view to show. That way, in all the places you are making variants of this behavior to show and hide views, you can just call that one function.